### ISSUE TITLE
Support Backbone 1.2

### ISSUE THREAD

**Issue by softcraft-development** [codebrew/backbone-rails#193]
Backbone 1.2 just came out:

https://github.com/jashkenas/backbone/releases/tag/1.2.0

It's been a while since backbone-rails has been touched; does anyone have an idea when the 1.2 upgrade will be available?

**Comment by softcraft-development** [comment]
I did a quick-and-dirty upgrade in my own fork: https://github.com/codebrew/backbone-rails/pull/194

I'm not sure if what I did follows your usual upgrade procedures, but it does seem to work well enough in my initial testing.

Note: in addition to rolling the gem version number, I changed the gem name in the gemspec to match the repo name. Without this, bundler failed when trying to install the gem from a github source.
